source/install/troubleshooting.rst

```diff
@@ -64,6 +64,8 @@ The following is a list of common error messages and solutions:
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 -  Message appears in blue bar on team site.
+-  To check the websocket connection, open the developer console in your browser and view the Network panel. Then look for a "websocket" connection to see if it's working. 
```

Comment:
Propose calling out that the screenshot is from Chrome? Otherwise looks  great! 

People have really struggled with this and this helps